lib/Conversion/HWToLLVM/HWToLLVM.cpp

```diff
@@ -675,9 +675,6 @@ void HWToLLVMLoweringPass::runOnOperation() {
   populateHWToLLVMTypeConversions(converter);
 
   LLVMConversionTarget target(getContext());
-  target.addLegalOp<UnrealizedConversionCastOp>();
-  target.addLegalOp<ModuleOp>();
-  target.addLegalDialect<LLVM::LLVMDialect>();
   target.addIllegalDialect<hw::HWDialect>();
```

Comment:
These were already implicit in the LLVMConversionTarget - no need to set them up again. For ModuleOp we don't need to mark it legal since we are doing a `applyPartialConversion`.  Not technically needed for the bump to work, the real change is modifying the test to run the `reconcile-unrealized-casts` afterwords.